keyboards/1upkeyboards/1up60hte/rules.mk

```diff
@@ -54,6 +16,5 @@ NKRO_ENABLE = yes		# USB Nkey Rollover - if this doesn't work, see here: https:/
 BACKLIGHT_ENABLE = yes  # Enable keyboard backlight functionality
 AUDIO_ENABLE = no
 RGBLIGHT_ENABLE = yes
-EXTRAFLAGS += -flto
```

Comment:
Why remove this?

Reply:
It was originally put in because bootmagic_enable was set to yes and firmware size was deemed too large I believe. 

What's in master:
* The firmware size is fine - 20228/28672 (70%, 8444 bytes free)

In master without extra flags:
 * The firmware size is fine - 22492/28672 (78%, 6180 bytes free)


Current change:
 * The firmware size is fine - 21894/28672 (76%, 6778 bytes free)

If we put the extraflags back in we get
 * The firmware size is fine - 19632/28672 (68%, 9040 bytes free)

Should I put it back in?

Reply:
Ah, okay.  That makes sense.

And no, if it has that much free space, reported by Travis, then it's fine.